### ISSUE TITLE
Do we have any script to generate aligned hairstyles with ./data/head_prior.obj

### ISSUE THREAD

**Issue by umariqb** [Vanessik/HAAR#6]
Hi @Vanessik @gfgb, 

I am trying to reproduce the training of HAAR. Is there any script available to generate the aligned pointclouds for USCHairSaloon and CT2Hair datasets?

Thanks!

**Comment by fighting-Zhang** [comment]
+1 😄

**Comment by Vanessik** [comment]
Our head is aligned with the scalp from USCHairSalon, so you could just use hairstyles after reading from .data to .ply as they are.  Regarding CT2Hair I don't have access to the alignment scripts, but have this data if needed.

**Comment by fighting-Zhang** [comment]
The current implementation of **Hairstyle Dataset** seems to only accept hair strands represented by exactly 100 points (shape (100, 3)). However, **the hair strand data generated by CT2Hair does not consistently have 100 points**. This creates a compatibility issue when attempting to use CT2Hair outputs with Hairstyle.

Should CT2Hair outputs be resampled or padded to 100 points before feeding into Hairstyle? If so, what is the recommended method?

**Comment by Vanessik** [comment]
@fighting-Zhang , yes you are right, completely forgot about a different number of points. Basically, you just resample each strand from N points to 100 or use for all hairstyles another number of points on each strand. Regarding the method, just a simple [interpolation function](https://pytorch.org/docs/stable/generated/torch.nn.functional.interpolate.htmll) with mode='linear' should be fine.
